### PR TITLE
Den Validation

### PR DESCRIPTION
**Pull request by Anna-Ivanova**

### CODE REVIEW

**Review by Anna-Ivanova — COMMENTED** [review]
просмотрено

**Review by Anna-Ivanova — COMMENTED** [review]
approved

**Comment by DEN3DEN3DEN** [comment]
add validation

**Comment by DEN3DEN3DEN** [comment]
hello

**Review by Anna-Ivanova — COMMENTED** [review]
How many comments I have to write.

File: index.html [hunk 1]
```diff
@@ -145,97 +145,116 @@ <h2 class="details_title">Customers</h2>
 
             <section class="forms">
                 <div class="editAddPeople">
-                    <form class="formEditAddPeople">
+                    <form class="form formEditAddPeople">
```
Comment (DEN3DEN3DEN) [inline index.html:148]: add class for validation

**Review by DEN3DEN3DEN — COMMENTED** [review]
add review

File: index.html (hunk 1 above)
Comment (Anna-Ivanova) [reply index.html:148]: ok

**Pull request closed without merging**